contentcuration/contentcuration/frontend/accounts/pages/accountDeleted/AccountDeleted.vue

```diff
@@ -2,13 +2,7 @@
 
   <MessageLayout
     :header="$tr('accountDeletedTitle')"
-  >
-    <template #back>
```

Comment:
In favor of consistency we could use the same aspect and label for all back buttons.

Reply:
Lets maintain these as raised buttons as you said in your PR description. Seems `MessageLayout` is exclusively used for these account related pages, so I think it's worth having it consistently rendered as a button.

Reply:
Understood! I have updated the message that was different from other continue to sign-in buttons as mentioned in the related issue.

Reply:
Reviewing what you did to resolve this, I realize I didn't originally clarify myself very well. I was thinking that `MessagingLayout` could be updated to use a `KButton` instead of `KRouterLink`, then you could remove some of these component slots like you originally had. Do you think that's possible? Definitely not a blocker though!

Reply:
Oh hehe, I think it wouldnt be possible because `MessageLayout` is used in other components that set by slot its own raised buttons, and if we update the back button of `MessageLayout` to be a `KButton` then in places like this we will have double raised primary buttons:

<img width="729" alt="image" src="https://user-images.githubusercontent.com/51239030/212045005-3c90a326-a5fb-4633-a60b-2d01280936f0.png">

Reply:
Sorry for the late reply @AlexVelezLl. That makes sense. I will get this reviewed and merged